lib/fog/ovirt/models/compute/server.rb

```diff
@@ -31,6 +31,8 @@ class Server < Fog::Compute::Server
         attribute :ips
         attribute :ha
         attribute :ha_priority
+        attribute :clone
```

Comment:
Good point. It seems like we could add this on top of the current patch pretty easily without issue, so I'm going to go ahead and accept this and we can add that later if desired.